### PR TITLE
0.6.0

### PR DESCRIPTION
**Pull request by mindplay-dk**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,9 +13,11 @@
             "mindplay\\sql\\": "src/"
         }
     },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
```
Comment (harikt) [inline composer.json]: why do you need these two?

Comment (mindplay-dk) [reply composer.json]: Because `unbox^2` still lives on an unstable branch :-)

Comment (harikt) [reply composer.json]: ok, why not use `2.*@dev` for that then? Just to confirm if you add the above lines does it helps people not to add minimum-stability flag on the root of the composer.json on their project?

Comment (mindplay-dk) [reply composer.json]: These flags _only_ apply at the project-level, e.g. only apply to this package during testing and development. We're releasing tomorrow, so I will probably remove them again then :-)

**Pull request merged**